src/data/builder-tools.js

```diff
@@ -339,6 +339,14 @@ export const Showcases = [
     getstarted: "/docs/get-started/blockfrost",
     tags: ["favorite", "http", "json", "hosted"],
   },
+  {
+    title: "StakePool Operator Scripts",
+    description: "Scripts to help manage your stake pool, send transactions, mint/burn tokens, etc. via the CLI.",
```

Comment:
"Hardware-Ledger/Trezor Support" would be a nice addition in the description.

Reply:
I can see from the preview that there's a lot more space to add the HW Wallet component to the description (I was judging by the typical line length in the JS file, which was misleading about the amount of space available).  I'll submit a revision that that element in it.